Assets/Scripts/Models/Inventory.cs

```diff
@@ -23,6 +21,7 @@ public class Inventory : IXmlSerializable, ISelectable, IContextActionProvider
     public string objectType = "Steel Plate";
     public int maxStackSize = 50;
     public float basePrice = 1f;
+    public string category = "inv_cat_none";
```

Comment:
Do you need to edit this field from Unity editor?
If not - make it property instead.
*Best practice - no public fields*